core/web/schema/type/spec.graphql

```diff
@@ -126,6 +126,8 @@ type BlockhashStoreSpec {
     waitBlocks: Int!
     lookbackBlocks: Int!
     blockhashStoreAddress: String!
+    trustedBlockhashStoreAddress: String
+    trustedBlockhashStoreBatchSize: Int!
```

Comment:
So `trustedBlockhashStoreBatchSize` is required but `trustedBlockhashStoreAddress` isn't? Should both be optional instead?

Reply:
Yeah this is just an accident, will make sure it isn't required.